.circleci/config.yml

```diff
@@ -38,24 +75,11 @@ jobs:
           path: storybook/
           destination: storybook
 
-  test:
-    working_directory: ~/mattermost-desktop
-    docker:
-      - image: electronuserland/builder:wine-chrome
-    steps:
-      - checkout
-      - run: apt-get update && apt-get -y install --no-install-recommends libgconf-2-4
-      - run: npm install
-      # https://github.com/SeleniumHQ/docker-selenium/issues/222#issuecomment-221307583
-      - run: DBUS_SESSION_BUS_ADDRESS=/dev/null xvfb-run npm test
-      - run: mkdir -p /tmp/test-results
-      - run: cp test-results.xml /tmp/test-results/
-      - store_test_results:
-          path: /tmp/test-results
-
 workflows:
   version: 2
   build_and_test:
     jobs:
-      - build
-      - test
```

Comment:
why you remove the test part?

Reply:
I merged testing and linting and changed the name to check (a bit more generic)